### DISCUSSION TITLE
Modularized :: New Camera :: Error parsing AU headers

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#338] [Q&A] [answered]
New 4K camera. New issue - LOL

I get this in the logs  a lot although the camera seems to be working fine:

`[2022-03-26 07:25:41] [ERROR   ] [viseron.components.ffmpeg.stream.BackEast] - [rtsp @ 0x55b7824cac40] Error parsing AU headers`

**Comment by gniknalu** [comment]
I kept messing with the camera's configuration until the error went away.

**Comment by roflcoopter** [comment, marked as answer]
Glad you sorted it out.

Errors like those are printed by FFMPEG, not much you can do about it really.
They shouldn't cause any issues tho.

You can add them to `ffmpeg_recoverable_errors` and they will be suppressed from the logs
